safe-global/safe-wallet-web
# feat: add claim button to header

## What it solves

Implements claim button in header

## How this PR fixes it

The token allocation element in the header now includes a "New allocation" button according to [the designs](https://www.figma.com/file/ptTs6lDBeUuLNySroJ5PiF/Web-Master-File?type=design&node-id=7969-133988&mode=design&t=6FeK1GEhF8jSaoo7-0). Clicking it will take you to the claiming app and, providing there is nothing to claim, it should not appear.

## How to test it

Open a Safe with an allocation and observe the new button. Claiming should make it dissappear and opening a Safe _without_ an allocation should not show it at all.

Clicking on the button should the following event: "Click on SEP5 allocation button".

## Screenshots

![image](https://github.com/safe-global/safe-wallet-web/assets/20442784/e4fb216d-936c-455e-b07e-0f62a07fbe2d)

![image](https://github.com/safe-global/safe-wallet-web/assets/20442784/2d977a6d-28ed-42b3-b17b-4b4518813f56)

## Checklist
